net-voip/captagent/captagent-6.3.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="CaptAgent:  100% Open-Source Packet Capture Agent for HEP"
+HOMEPAGE="https://sipcapture.org/"
+SRC_URI="https://github.com/sipcapture/captagent/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ipv6 mysql pcre redis ssl uv"
+
+PATCHES=(
+	# https://github.com/sipcapture/captagent/pull/239 (should be accepted).
+	"${FILESDIR}/${P}-gcc10.patch"
+	# Already upstreamed for next version.
+	"${FILESDIR}/${P}-configure.patch"
+)
+
+DEPEND="dev-libs/json-c
+	net-libs/libpcap
+
```

Comment:
Could we drop this empty line from here.

Reply:
Done.  I just prefer having a blank line between the unconditional and conditional depends, but that's personal preference, if the policy is to not have it, so be it.